hs_core/views/serializers.py

```diff
@@ -0,0 +1,75 @@
+__author__ = 'Pabitra'
+
+from hs_core import hydroshare
+from django.contrib.auth.models import Group, User
+from rest_framework import serializers
+from .utils import authorize, validate_json, validate_user_name, validate_group_name
+from hs_core.hydroshare import utils
```

Comment:
Import ordering is a mess all over the project. For new modules added I'd like to see the import ordering improved. Imports should be listed from wide to narrow: Python standard library, external libraries, Django, followed by imports from hydroshare itself last. The groups should be separated, usually.

This just helps following the module and what is in play a lot easier.